setup.cfg

```diff
@@ -29,3 +29,20 @@ previous = true
 [flake8]
 max-line-length = 88
 extend-ignore = E203
+
+# mypy related settings
+[mypy]
+plugins = sqlalchemy.ext.mypy.plugin
```

Comment:
The reason I'm commenting above, is that down here it's _not_ activating the plugin that was installed from  `sqlalchemy2-stubs` but rather using the one in `SQLAlchemy[mypy]`.

Reply:
no more sqlalchemy2-stub, solved